build.gradle

```diff
@@ -23,33 +23,43 @@ buildscript {
 def hadoopVersion = "3.1.1"
 
 ext.deps = [
-    hadoop: [
-        "common": "org.apache.hadoop:hadoop-common:${hadoopVersion}",
-        "common_test": "org.apache.hadoop:hadoop-common:${hadoopVersion}:tests",
-        "hdfs": "org.apache.hadoop:hadoop-hdfs:${hadoopVersion}",
-        "hdfs_client": "org.apache.hadoop:hadoop-hdfs-client:${hadoopVersion}",
-        "hdfs_test": "org.apache.hadoop:hadoop-hdfs:${hadoopVersion}:tests",
-        "mapreduce_client_core": "org.apache.hadoop:hadoop-mapreduce-client-core:${hadoopVersion}",
-        "minicluster": "org.apache.hadoop:hadoop-minicluster:${hadoopVersion}",
-        "yarn_api": "org.apache.hadoop:hadoop-yarn-api:${hadoopVersion}",
-        "yarn_client": "org.apache.hadoop:hadoop-yarn-client:${hadoopVersion}",
-        "yarn_common": "org.apache.hadoop:hadoop-yarn-common:${hadoopVersion}",
-        "yarn_server_test": "org.apache.hadoop:hadoop-yarn-server-tests:${hadoopVersion}:tests"
-    ],
-    external: [
-        "avro": "org.apache.avro:avro:1.8.2",
-        "guava": "com.google.guava:guava:16.0.1",
-        "jackson_databind": "com.fasterxml.jackson.core:jackson-databind:2.8.3",
-        "jackson_dataformat_yaml": "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.9.6",
-        // Only needed by Hadoop test classes
-        "junit": "junit:junit:4.12",
-        "objenesis": "org.objenesis:objenesis:2.6",
-        "py4j": "net.sf.py4j:py4j:0.8.2.1",
-        "sshd": "org.apache.sshd:sshd-core:1.1.0",
-        "testng": "org.testng:testng:6.4",
-        "text": "org.apache.commons:commons-text:1.4",
-        "zip4j": "net.lingala.zip4j:zip4j:1.3.2"
-    ]
+  azkaban: [
+    "az_core": "com.linkedin.azkaban:az-core:3.58.0",
+    "azkaban_common": "com.linkedin.azkaban:azkaban-common:3.58.0",
+    "azkaban_plugins": "com.linkedin.azkaban:az-hadoop-jobtype-plugin:3.58.0",
+    "azkaban_security": "com.linkedin.azkaban:azkaban-hadoop-security-plugin:3.58.0"
```

Comment:
Can we use the full names `az_hadoop_jobtype_plugins` and `azkaban_hadoop_security_plugin` as our variable names? Azkaban has other types of plugins besides jobtype plugins, and also other types of security plugins.

Reply:
sure, fixed